packages/dmn-js-decision-table/src/features/column-resize/components/ResizeColumn.js

```diff
@@ -0,0 +1,104 @@
+import { Component } from 'inferno';
+
+import { inject } from 'table-js/lib/components';
+
+import {
+  closest as domClosest
+} from 'min-dom';
+
+import { is } from 'dmn-js-shared/lib/util/ModelUtil';
+
+
+export default class ResizeColumn extends Component {
+
+  constructor(props, context) {
+    super(props, context);
+
+    inject(this);
+  }
+
+  handleMouseDown = event => {
+    const cell = domClosest(event.target, 'th', false);
+
+    startResize({
+      node: cell,
+      event,
+      minWidth: this.props.minWidth || 150,
+      onEnd: this.saveWidth
+    });
+  }
+
+  saveWidth = newWidth => {
+    const { col } = this.props;
+
+    const update = {};
+
+    if (is(col, 'dmn:DecisionTable')) {
+      update.annotationsWidth = newWidth;
+    } else {
+      update.width = newWidth;
+    }
+
+    this.modeling.updateProperties(col, update);
+  }
+
+  render() {
+    return (
+      <div
+        onMouseDown={ this.handleMouseDown }
+        className="resize-column-handle"
+        title={ this.translate('Resize') }
+      />
+    );
+  }
+}
+
+ResizeColumn.$inject = [ 'sheet', 'changeSupport', 'translate', 'modeling' ];
+
+
+
+// helper
```

Comment:
The new-line spacing is a bit weird here, maybe you can use one new-line from above to insert here.

Reply:
I'll use a single empty line spacing, right?

Reply:
jup

Reply:
// helpers ////////// 🥇

Reply:
🙅 ⛔ 🚫